internal/nix/input.go

```diff
@@ -50,46 +76,62 @@ func InputFromString(raw string, locker lock.Locker) *Input {
 	return &Input{*u, locker, raw, ""}
 }
 
+// InputFromProfileItem sets the raw Input as the `item`'s unlockedReference i.e.
+// the flake reference and output attribute path used at install time.
 func InputFromProfileItem(item *NixProfileListItem, locker lock.Locker) *Input {
 	return InputFromString(item.unlockedReference, locker)
 }
 
-func (i *Input) IsLocal() bool {
+// isLocal specifies whether this input is a local flake.
+// Usually, this is of the form: `path:./local_flake_subdir#myPackage`
+func (i *Input) isLocal() bool {
 	// Technically flakes allows omitting the scheme for local absolute paths, but
 	// we don't support that (yet).
 	return i.Scheme == "path"
 }
 
-func (i *Input) IsDevboxPackage() bool {
+// isDevboxPackage specifies whether this input is a nix package defined in a devbox.json config.
```

Comment:
This is not specific enough. This specifically checks if it's a devbox package as defined by our own syntax (canonical@version). A "nix" package can be a flake or a legacy attribute path.

Reply:
sounds good, I'll update.